docker-compose.yml

```
@@ -790,6 +790,8 @@ services:
       KSQL_SSL_KEYSTORE_LOCATION: /etc/kafka/secrets/kafka.ksqlDBserver.keystore.jks
       KSQL_SSL_KEYSTORE_PASSWORD: confluent
       KSQL_SSL_KEY_PASSWORD: confluent
+      KSQL_SSL_ENABLED_PROTOCOLS: "TLSv1.3,TLSv1.2"
```

@javabrett in light of the upcoming change in 6.1.0, the suggestion below will help for upkeep (specifically, in 6.1.0, this will help us flag for removal, and inform users)

```suggestion
      # Before v6.1.0: disabling TLSv1.x is required 
      KSQL_SSL_ENABLED_PROTOCOLS: "TLSv1.3,TLSv1.2"
```